test/integration/application-log-assersion/karma.conf.js

```diff
@@ -0,0 +1,67 @@
+const customCucumberReporter = require("./karma-reporter/index");
+const customKarmaUI5 = require("../../../lib/index");
+
+module.exports = function(config) {
+	"use strict";
+
+	require("../karma-base.conf")(config);
+	config.set({
+		plugins: [
+			"karma-chrome-launcher",
+			"karma-coverage",
+			customKarmaUI5,
+			customCucumberReporter
+		],
+
+		ui5: {
+			type: "application",
+			mode: "html",
+			testpage: "webapp/test/GherkinTestRunner.html",
+			url: "http://localhost:" + config.localUI5ServerPort,
+			logAssertions: true
+		},
+
+		frameworks: ["ui5"],
+
+		preprocessors: {
+			"{webapp,webapp/!(test)}/*.js": ["coverage"]
+		},
+		cucumberReporter: {
+			out: "application-log-assersion/cucumber.json",
```

Comment:
Should be asser**t**ion (also the folder should be renamed)
```suggestion
			out: "application-log-assertion/cucumber.json",
```